### PR TITLE
[KIECLOUD-358] sha retrieval, disconnectedMode, and csv version changes

### PR DESCRIPTION
**Pull request by tchughesiv**
`spec.useImageTags` is `false` by default. Setting to true will disable SHA digest utilization in favor of image tags for product versions 7.7.0 and greater. 

Signed-off-by: tchughesiv <tchughesiv@gmail.com>

### CODE REVIEW

File: deploy/ui/form.json [hunk 1]
```diff
@@ -47,6 +47,14 @@
             }
           ]
         },
+        {
+          "label": "Disable Disconnected Mode",
```
Comment (bmozaffa) [inline deploy/ui/form.json]: Shouldn't we just say "Use Image Tags" here and explain the consequences in the description as you do?

File: tools/csv-gen/csv-gen.go [hunk 2]
```diff
@@ -239,6 +243,12 @@ func main() {
 						Path:         "upgrades.minor",
 						XDescriptors: []string{"urn:alm:descriptor:com.tectonic.ui:booleanSwitch"},
 					},
+					{
+						Description:  "Set true to disable disconnected install mode and, instead, enable image tags. Image Tags are disabled by default. This will leverage image tags instead of the image SHA digests.",
+						DisplayName:  "Disable Disconnected Mode",
```
Comment (bmozaffa) [inline tools/csv-gen/csv-gen.go]: Same question here

File: deploy/ui/form.json (hunk 1 above)
Comment (tchughesiv) [reply deploy/ui/form.json]: done

File: tools/csv-gen/csv-gen.go (hunk 2 above)
Comment (tchughesiv) [reply tools/csv-gen/csv-gen.go]: done

**Pull request merged**